kernel/poll.c

```diff
@@ -68,6 +68,7 @@ static inline int is_condition_met(struct k_poll_event *event, u32_t *state)
 		return 0;
 	default:
 		__ASSERT(0, "invalid event type (0x%x)\n", event->type);
+		break;
```

Comment:
MISRA C really demands the presence of a break after a default: case where it's a noop?  Sigh.  I mean, harmless and all, but still...

Reply:
This break is noop only if you always put default case at the end. I think that it is safer to just a the break.